### PR TITLE
Playground js molecule field

### PR DESCRIPTION
**Pull request by juanmaguitar**
Use of new `demo` folder to allow import of external dependencies
Use cases of `MoleculeField` component integrated with `FormInput` & `AtomTextarea`
Also use case of `MoleculeField` inside of a dynamic form with validation rules

### CODE REVIEW

File: demo/molecule/field/demo/package.json
Comment (davidbarna) [inline demo/molecule/field/demo/package.json]: use major/exact versions

File: demo/molecule/field/demo/FormWithFormik/index.js
Comment (davidbarna) [inline demo/molecule/field/demo/FormWithFormik/index.js]: Were are you using `Field` ?

File: demo/molecule/field/demo/MoleculeFieldTextarea/hoc/WithCharacterCount.js
Comment (davidbarna) [inline demo/molecule/field/demo/MoleculeFieldTextarea/hoc/WithCharacterCount.js]: Isn't this should be as a service of the field instead of demo ?

**Review by davidbarna — DISMISSED** [review]

**Comment by jcger** [comment]
Enablers will discuss about if it's ok to use formik

File: demo/molecule/field/demo/index.scss
Comment (jcger) [inline demo/molecule/field/demo/index.scss]: you need to add the input css too

File: demo/molecule/field/demo/package.json
Comment (jcger) [inline demo/molecule/field/demo/package.json]: use exact version please, you can npm install with -DE (save dev exact) or -SE (save exact) in order do not add the ^

File: demo/molecule/field/demo/index.js
Comment (jcger) [inline demo/molecule/field/demo/index.js]: you are not using this, right?

File: demo/molecule/field/demo/FormWithFormik/index.js
Comment (jcger) [inline demo/molecule/field/demo/FormWithFormik/index.js]: you could use the @schibstedspain/sui-atom-button

File: demo/molecule/field/demo/index.scss
Comment (juanmaguitar) [reply demo/molecule/field/demo/index.scss]: Added!

**Review by jcger — DISMISSED** [review]

**Review by davidbarna — DISMISSED** [review]

**Comment by juanmaguitar** [comment]
Going to wait for the approval of the `MoleculeFieldInput` (https://github.com/SUI-Components/sui-components/pull/507) and its integration in this demo, to move on this PR

**Comment by juanmaguitar** [comment]
Waiting for #516 to link proper `MoleculeTextareaField` in this demo

**Review by jcger — DISMISSED** [review]

**Review by jcger — DISMISSED** [review]

**Review by jcger — APPROVED** [review]

**Review by carlosvillu — APPROVED** [review]

**Pull request merged**
